modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -10,7 +10,7 @@ Runtime templates related to Partner Manager are upgraded when:
 * Functional and performance defects are resolved in the template
 * Flow optimizations are implemented
 
-When new runtime templates are available, the *Manage deployments* button is enabled with these options:
+When new runtime templates are available, the *Manage deployments* button changes color and shows a tooltip to indicate a new version is available. To take advantage of the new features, click *Manage deployments* and select one of these options:
```

Comment:
Please remove 'changes color and' as there is no change of color at all now. The Manage deployments button looks the same when there is new runtime version available or not.